langchain4j-core/src/test/java/dev/langchain4j/rag/DefaultRetrievalAugmentorTest.java

```diff
@@ -64,12 +86,12 @@ void should_augment_user_message__multiple_queries_multiple_retrievers(Executor
         ContentInjector contentInjector = spy(new TestContentInjector());
 
         RetrievalAugmentor retrievalAugmentor = DefaultRetrievalAugmentor.builder()
-                .queryTransformer(queryTransformer)
-                .queryRouter(queryRouter)
-                .contentAggregator(contentAggregator)
-                .contentInjector(contentInjector)
-                .executor(executor)
-                .build();
+            .queryTransformer(queryTransformer)
```

Comment:
Can we change the code style setting so that existing code style is preserved? Now every PR has lots of reformatting...

Reply:
Let's tune https://github.com/langchain4j/langchain4j/blob/main/.editorconfig

Reply:
OK, but let's keep existing code style to not loose git history